index.html

```diff
@@ -551,7 +551,40 @@ <h4>South Asian (Indic script) languages</h4>
 
         </section>
 
-         </section>
+		<section id="cjk-ivs">
+			<h4>East Asian languages</h4>
```

Comment:
I wouldn't bury this under "orthographic variation", since it is not really a "spelling" variation. It's really an encoding variation. I would promote this up a level and put it just after East Asian Width. All of the other subsections of "Problems with Determining Equivalence" are the specific problem, so I'd title it something like:

```suggestion
       <h4>Ideographic Variation Sequences</h4>
```